content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -719,6 +719,66 @@ webhooks:
 
 The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
 
+### Matching requests: matchConditions
+
+{{< feature-state state="alpha" for_k8s_version="v1.27" >}}
+
+{{< note >}}
+Match conditions requires the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`AdmissionWebhookMatchConditions` to be explicitly enabled before this feature can be used.
+{{< /note >}}
+
+Match conditions can be added to webhooks for fine-grained request filtering (in conjunction with
+match rules, objectSelectors and namespaceSelectors) using
+[CEL expressions](/docs/reference/using-api/cel/). All match conditions must evaluate to true for
+the webhook to be called.
+
+Here are a couple examples of match conditions:
+
+```yaml
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+webhooks:
+- name: my-webhook.example.com
+  matchPolicy: Equivalent
+  rules:
+  - operations: ['CREATE','UPDATE']
+    apiGroups: ['*']
+    apiVersions: ['*']
+    resources: ['*']
+  matchConditions:
+  - name: 'exclude-leases' # Each match condition must have a unique name
+    expression: '!(request.resource.group == "coordination.k8s.io" && resource.resource == "leases")' # Match non-lease resources.
+  - name: 'exclude-kubelet-requests'
+    expression: '!("system:nodes" in request.userInfo.groups)' # Match requests made by non-node users.
+  - name: 'breakglass'
+    # Skip requests by users with the 'breakglass' permission on my-webhook.
+    expression: 'authorizer.group('admissionregistration.k8s.io').resource('validatingwebhookconfigurations').name('my-webhook.example.com').check('breakglass').denied()'
```

Comment:
If feasible, and this is a valid manifest, make this use a `{{< codenew >}}` shortcode.